migrations/versions/f0258425e851_audit_and_timestamp_indecies.py

```diff
@@ -0,0 +1,263 @@
+"""audit_and_timestamp_indecies
+
+Revision ID: f0258425e851
+Revises: 32145d0959ae
+Create Date: 2021-09-03 10:08:21.530412
+
+"""
+
+# revision identifiers, used by Alembic.
+revision = 'f0258425e851'
+down_revision = '32145d0959ae'
+
+from alembic import op
+import sqlalchemy as sa
+import sqlalchemy_utils
+
+import app
+import app.extensions
+
+
+
+def upgrade():
+    """
+    Upgrade Semantic Description:
+        ENTER DESCRIPTION HERE
+    """
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table('annotation', schema=None) as batch_op:
+        batch_op.create_index(batch_op.f('ix_annotation_created'), ['created'], unique=False)
+        batch_op.create_index(batch_op.f('ix_annotation_updated'), ['updated'], unique=False)
+
+    with op.batch_alter_table('annotation_keywords', schema=None) as batch_op:
+        batch_op.create_index(batch_op.f('ix_annotation_keywords_created'), ['created'], unique=False)
+        batch_op.create_index(batch_op.f('ix_annotation_keywords_updated'), ['updated'], unique=False)
+
+    with op.batch_alter_table('asset', schema=None) as batch_op:
+        batch_op.create_index(batch_op.f('ix_asset_created'), ['created'], unique=False)
+        batch_op.create_index(batch_op.f('ix_asset_updated'), ['updated'], unique=False)
+
+    with op.batch_alter_table('asset_group', schema=None) as batch_op:
+        batch_op.create_index(batch_op.f('ix_asset_group_created'), ['created'], unique=False)
+        batch_op.create_index(batch_op.f('ix_asset_group_updated'), ['updated'], unique=False)
+
+    with op.batch_alter_table('asset_group_sighting', schema=None) as batch_op:
+        batch_op.create_index(batch_op.f('ix_asset_group_sighting_created'), ['created'], unique=False)
+        batch_op.create_index(batch_op.f('ix_asset_group_sighting_updated'), ['updated'], unique=False)
+
+    with op.batch_alter_table('audit_log', schema=None) as batch_op:
+        batch_op.add_column(sa.Column('duration', sa.Float(), nullable=True))
```

Comment:
There is something missing here. The max length has been dropped from the user_email and length in the model definition but the change is not reflected here. 
This is what is causing the test failures on github I think. 
houston=# \d audit_log
                          Table "public.audit_log"
   Column    |            Type             | Collation | Nullable | Default 
-------------+-----------------------------+-----------+----------+---------
 created     | timestamp without time zone |           | not null | 
 updated     | timestamp without time zone |           | not null | 
 guid        | uuid                        |           | not null | 
 module_name | character varying(50)       |           |          | 
 item_guid   | uuid                        |           |          | 
 user_email  | character varying(120)      |           | not null | 
 message     | character varying(240)      |           |          | 
 audit_type  | character varying           |           | not null | 
 duration    | double precision            |           |          | 
This is on my local machine where the tests work perfectly. No I don't understand either.